### ISSUE TITLE
Modify requirements.txt to accomodate changes in fasm 

### ISSUE THREAD

**Issue by Mattia9875** [SymbiFlow/symbiflow-examples#124]
In ```xc7/requirements.txt``` 
substitute ```git+https://github.com/symbiflow/fasm``` 
with ```--no-use-pep517 git+https://github.com/SymbiFlow/fasm.git#egg=fasm```

as of [issue](https://github.com/SymbiFlow/fasm/issues/60) over on /symbiflow/fasm

**Comment by litghost** [comment]
Now that `fasm` is on PyPi, the xc7/requirements can just remove the `fasm` requirement completely now that it is in PyPi.

**Comment by Mattia9875** [comment]
You mean that in the sense that i could do it on my system? or that is the general way to fix it?

**Comment by litghost** [comment]
> You mean that in the sense that i could do it on my system? or that is the general way to fix it?

As in you/I could submit a PR to fix this?

**Comment by Mattia9875** [comment]
Yes, though Tim responded on the issue I cited saying that he is currently working on in so I don't know how this issue will stand
